business-registry-dashboard/app/composables/useBrdModals.ts

```diff
@@ -70,6 +70,10 @@ export const useBrdModals = () => {
     modal.open(ModalManageBusiness, { business })
   }
 
+  function openAuthEmailSent (email: string) {
```

Comment:
This will open the email auth sent modal if a user clicks the 'resend email' table action